src/lib/workspace/WorkspaceRouter.tsx

```diff
@@ -211,6 +212,28 @@ export function WorkspaceRouter({
               </WorkspaceContainer>
             )}
           />
+          <Route
+            path={[
+              `${path}/:studyId/:analysisId/fullscreen-map`,
+              `${path}/:studyId/new/fullscreen-map`,
+            ]}
```

Comment:
Could this multiple path matching be used to DRY up the regular analysis routes? (`:analysisId` and `new`)